tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +54,8 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result == False
```

Comment:
Great assert! It would also be a good idea to make sure the inventory length is still 3 and that nothing was removed accidentally.

Reply:
I'll keep this in mind and include more necessary asserts in the future!